src/main/java/com/mewna/catnip/internal/CatnipImpl.java

```diff
@@ -413,96 +382,6 @@ private void injectSelf() {
         taskScheduler.catnip(this);
     }
     
-    private void codecs() {
```

Comment:
:crab: CODECS ARE GONE :crab:

This means you can delete those from the util package now.

Reply:
deleted them codecs